app/services/analytics.rb

```diff
@@ -30,4 +30,50 @@ def ahoy
   def uuid
     user.uuid
   end
+
+  # rubocop:disable Metrics/LineLength
```

Comment:
I disabled line length here because leaving everything as one-liners makes it easy to use my editor to sort these constants alphabetically